### ISSUE TITLE
Community groups are being include in leaderboards/scores and showing up on normal sorting options

### ISSUE THREAD

**Issue by TemetN** [Metaculus/metaculus#908]
As topic, questions from community groups which are not supposed to be visible in questions orting or impact the leaderboards are doing so. In addition some have already closed and impacted scoring, and may need to be removed retroactively from the leaderboards (I honestly have no idea how that is handled).

**Comment by SylvainChevalier** [comment]
@TemetN sorry, I'm having trouble following. Could you point to specific questions that are problematic, and the leaderboards that are impacted?

**Comment by TemetN** [comment]
@SylvainChevalier Sure, basically questions which are not approved/designed for the normal sorts/leaderboards are showing up and giving points (I was told they're called community groups, which I admit I have no idea if it's correct or not.) Some examples (pulled specifically from new) are 'O Brasil melhorará sua posição no ranking de índice de pobreza (do G20) até Setembro de 2026?' and 'Will the Shoreline Park pickleball courts be started by week of Oct 28?' (as well as forecasts from groups such as Dave Mather's Community and Brown Forecasting Team).

Basically after we migrated questions that did not previous show up on new or closing soon did so, I reported this under the new site thread and got told that it was 'community groups' and they'd definitely have to be removed. Went and checked and noticed it wasn't here, so I moved it over. I should also note that the scoring thing is something someone mentioned when I asked, they said they were currently giving points so that's at second hand.
